src/jobs/SendPayloadJob.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace viget\phonehome\jobs;
+
+use craft\queue\BaseJob;
+use viget\phonehome\PhoneHome;
+
+/**
+ * Send Payload Job queue job
+ */
+class SendPayloadJob extends BaseJob
+{
+    function execute($queue): void
+    {
+        PhoneHome::getInstance()->phoneHome->sendPayload();
+    }
+
+    protected function defaultDescription(): ?string
+    {
+        return 'Phoning Home';
```

Comment:
☎️ Phoning Home
👽 Phoning Home

Options...

Reply:
That would be fun... I'm a little leery of emoji in SQL databases. It's probably not a huge issue... but I'd rather not think about it.

Should be resolved fully for Craft 5 (they're tweaking the default charset to be utf8mb4).

https://github.com/craftcms/cms/issues/12935#issuecomment-1495108327